packages/gestalt/src/SearchGuide.tsx

```diff
@@ -0,0 +1,191 @@
+
+  // When using both forwardRef and innerRef, React.useimperativehandle() allows a parent component
+  // that renders <SearchGuide ref={inputRef} /> to call inputRef.current.focus()
+  // @ts-expect-error - TS2322 - Type 'HTMLButtonElement | null' is not assignable to type 'HTMLButtonElement'.
+  useImperativeHandle(ref, () => innerRef.current);
+  const { isFocusVisible } = useFocusVisible();
+
+  const buttonClasses = isInVRExperiment
+    ? classnames(styles.searchguideVr, touchableStyles.tapTransition, {
+        [styles[`color${color}`]]: !selected,
+        [focusStyles.hideOutline]: !isFocusVisible,
+        [styles.vrFocused]: isFocusVisible,
+        [styles.selectedVr]: selected,
+      })
+    : classnames(styles.searchguide, touchableStyles.tapTransition, [styles[`color${color}`]], {
+        [styles.selected]: selected,
+        [focusStyles.hideOutline]: !isFocusVisible && !selected,
+        [focusStyles.accessibilityOutline]: isFocusVisible,
+      });
+  const childrenDivClasses = classnames(styles.childrenDiv);
+
+  const textComponent = (
+    <TextUI
+      align="center"
+      color={isInVRExperiment && selected ? 'inverse' : 'dark'}
+      overflow="noWrap"
+    >
+      {text}
+    </TextUI>
+  );
+  const thumbnailVariant = thumbnail && (
+    <Flex alignItems="center" gap={{ row: 2, column: 0 }} justifyContent="center">
+      {'avatar' in thumbnail && (
+        <Box aria-hidden marginStart={isInVRExperiment ? 2 : undefined} minWidth={40}>
+          {cloneElement(thumbnail.avatar, { size: 'fit' })}
+        </Box>
+      )}
+      {'avatarGroup' in thumbnail && (
+        <Box aria-hidden marginStart={isInVRExperiment ? 2 : undefined} minWidth={40}>
+          {cloneElement(thumbnail.avatarGroup, { size: 'sm' })}
+        </Box>
+      )}
+      {'image' in thumbnail && (
+        <div
+          className={
+            isInVRExperiment
+              ? classnames({
+                  [styles.imageDivVr]: !selected,
+                  [styles.selectedImageDivVr]: selected,
+                })
+              : styles.imageDiv
+          }
+        >
+          {cloneElement(thumbnail.image, { fit: 'cover' })}
+        </div>
+      )}
+      {'icon' in thumbnail && <Box marginStart={3}>{cloneElement(thumbnail.icon)}</Box>}
+      <Box marginEnd={3}>{textComponent}</Box>
+      {expandable ? <Icon accessibilityLabel="" color="dark" icon="arrow-down" size={12} /> : null}
+    </Flex>
+  );
+  const defaultVariant = (
+    <Box paddingX={5}>
+      <Flex alignItems="center" gap={{ row: 2, column: 0 }} justifyContent="center">
+        {textComponent}
+        {expandable ? (
+          <Icon accessibilityLabel="" color="dark" icon="arrow-down" size={12} />
+        ) : null}
+      </Flex>
+    </Box>
+  );
+  return (
+    <button
+      ref={innerRef}
+      aria-controls={accessibilityControls}
+      aria-expanded={accessibilityExpanded}
+      aria-haspopup={accessibilityHaspopup || expandable}
```

Comment:
for selected, we need accessibility, check what we did in ToggleButton.

![Screenshot by Dropbox Capture](https://github.com/user-attachments/assets/d303f0cc-7f00-4e09-9e5c-075b3ad0a724)